pkgs/top-level/python-packages.nix

```diff
@@ -1159,6 +1159,28 @@ in {
     };
   };
 
+  argcomplete = buildPythonPackage rec {
```

Comment:
Python modules have to be in the directory `pkgs/development/python-modules/` as suggested [here](http://nixos.org/nixpkgs/manual/#contributing-guidelines).